integration_tests/test_commit_format.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Test the commit message format."""
+
+import os
+
+from utils import get_cmd_output
+
+COMMIT_TITLE_MAX_LEN = 50
+COMMIT_BODY_LINE_MAX_LEN = 72
+BASE_BRANCH = os.environ['BUILDKITE_PULL_REQUEST_BASE_BRANCH']
+
+
+def test_commit_format():
+    """
+    Checks commit message format for the current PR's commits.
+
+    Checks if commit messages follow the 50/72 git commit rule
+    and if commits are signed.
+    """
+    # Get hashes of PR's commits in their abbreviated form for
+    # a prettier printing.
+    shas_cmd = "git log --no-merges --pretty=%h --no-decorate " \
```

Comment:
`git` won't be available in the container until v6 is published

Reply:
Sorry, I was under the impression it was updated in the last PR from there.

Reply:
The container image still has to be built and pushed to the Docker registry (sounds like something we could automate :thinking: but later); in the meantime I did that and v6 should be available now